pkg/nsx/services/staticroute/staticroute.go

```diff
@@ -133,3 +133,19 @@ func (service *StaticRouteService) ListStaticRoute() []model.StaticRoutes {
 	}
 	return staticRouteSet
 }
+
+func (service *StaticRouteService) Cleanup() error {
+	staticRouteSet := service.ListStaticRoute()
+	description := "cleanup staticroute"
```

Comment:
using the same log only print different parameter seems a little strange, could we still print what it is doing in log, eg: "removing static route" in 140

Reply:
done